inspirehep/modules/migrator/tasks.py

```diff
@@ -378,8 +337,11 @@ def create_record(record, force=True, dry_run=False):
     else:
         json = hep.do(record)
 
-    if dry_run:
-        return errors, json
+    if '$schema' in json:
+        json['$schema'] = url_for(
+            'invenio_jsonschemas.get_schema',
+            schema_path="records/{0}".format(json['$schema'])
+        )
```

Comment:
This code was actually just moved from https://github.com/inspirehep/inspire-next/pull/1160/files#diff-2b6424df440e3e9cba58ea4a4cab141aL272 as I thought it was better to have here (so that `create_record` does create the full record)